code/modules/animation/AnimationLibrary.dm

```diff
@@ -617,6 +617,24 @@ proc/muzzle_flash_any(var/atom/movable/A, var/firing_angle, var/muzzle_anim, var
 	icon_state = "shine"
 	plane = PLANE_OVERLAY_EFFECTS
 
+/obj/particle/cryo_sparkle
+	icon = 'icons/effects/chemistry_effects.dmi'
+	icon_state = "cryo-1"
+	plane = PLANE_OVERLAY_EFFECTS
+
+	New()
+		icon_state = pick("cryo-1", "cryo-2", "cryo-3", "cryo-4") //slightly different timings on these to give a less static look
```

Comment:
consider just having the same sprite (less work) but offsetting horizontally for some of these, so it's less 'tiled'